includes/archive/class-option.php

```diff
@@ -0,0 +1,126 @@
+<?php
+/**
+ * Option class.
+ *
+ * @link       https://www.boldgrid.com
+ * @since      SINCEVERSION
+ *
+ * @package    Boldgrid\Backup
+ * @subpackage Boldgrid\Backup\Archive
+ * @copyright  BoldGrid
+ * @author     BoldGrid <support@boldgrid.com>
+ */
+
+namespace Boldgrid\Backup\Archive;
+
+/**
+ * Class: Option
+ *
+ * This class is used to manage the boldgrid_backup_backups option.
+ *
+ * In it's first implementation, each entry in the array represents a single backup, and has an id and
+ * a filename. For examples, please see: https://pastebin.com/Wuey2zvP
+ *
+ * @since SINCEVERSION
+ */
+class Option {
+	/**
+	 * The option name storing backups.
+	 *
+	 * @since SINCEVERSION
+	 * @access private
+	 * @var string
+	 */
+	private $option = 'boldgrid_backup_backups';
+
+	/**
+	 * Get all our backups.
+	 *
+	 * @since SINCEVERSION
+	 *
+	 * @return array
+	 */
+	public function get_all() {
+		return get_option( $this->option, [] );
```

Comment:
Arrays should be declared with long array syntax per WordPress coding standards
https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#declaring-arrays

Reply:
See similar comment above.